spec/unit/puppet/provider/registry_value_spec.rb

```diff
@@ -88,15 +98,16 @@
 
   describe "#destroy" do
     let (:default_path) { path = "hklm\\#{puppet_key}\\#{subkey_name}\\" }
-    let (:path) { path = "hklm\\#{puppet_key}\\#{subkey_name}\\#{SecureRandom.uuid}" }
+    let (:valuename) { SecureRandom.uuid }
+    let (:path) { path = "hklm\\#{puppet_key}\\#{subkey_name}\\#{valuename}" }
     def create_and_destroy(path, reg_type, data)
       reg_value = type.new(:path => path,
         :type => reg_type,
         :data => data,
         :provider => described_class.name)
       already_exists = reg_value.provider.exists?
       already_exists.should be_falsey
-
+#require 'pry'; binding.pry
```

Comment:
Whoopsy daisy...

Reply:
No idea what you're talking about :-)  Done.